src/masternodes/govvariables/attributes.cpp

```diff
@@ -270,6 +288,14 @@ static ResVal<int32_t> VerifyPositiveInt32(const std::string& str) {
     return {int32, Res::Ok()};
 }
 
+static ResVal<CAttributeValue> VerifyPositiveInteger(const std::string& str) {
+    int32_t int32;
+    if (!ParseInt32(str, &int32)) {
```

Comment:
Make `int32` a `uin32_t` in the first place and call `ParseUInt32` to avoid casting `int32` to a `uint32_t` at the end of this call.